tables/tests/test_vlarray.py

```diff
@@ -1323,8 +1324,8 @@ def test05b_VLStringAtom(self):
         # Modify values
         vlarray[0] = "as4"
         vlarray[1] = "aaanc"
-        self.assertRaises(ValueError, vlarray.__setitem__, 1, "shrt")
-        self.assertRaises(ValueError, vlarray.__setitem__, 1, "toolong")
+        #self.assertRaises(ValueError, vlarray.__setitem__, 1, "shrt")
+        #self.assertRaises(ValueError, vlarray.__setitem__, 1, "toolong")
 
```

Comment:
Make sure that you add new tests for the assignation of values that are of different length than the old values.

Reply:
Also, do not forget to add an entry abut this new feature in the CHANGELOG for PyTables 4.0.